modules/openapi-generator/src/main/resources/dart2/api_client.mustache

```diff
@@ -1,16 +1,16 @@
 {{>header}}
 {{>part_of}}
 class ApiClient {
-  ApiClient({this.basePath = '{{{basePath}}}'}) {
+  ApiClient({this.basePath = '{{{basePath}}}', HttpBearerAuth? httpBearerAuth}) {
```

Comment:
Maybe add `{{#isBasicBearer}}` also here? It's kinda weird to have an constructor parameter without ausage

Reply:
nice